Resources/views/BreadCrumb/breadCrumb.html.twig

```diff
@@ -2,7 +2,10 @@
   {% if breadcrumbs.count %}
     <ol class="breadcrumb" itemscope itemtype="https://schema.org/breadcrumb">
       <li class="breadcrumb-item breadcrumb-item-back">
-        <a href="/" data-button-previous="back" class="breadcrumb-link-back"><i class="fas fa-chevron-left fa-sm mr-2"></i><span class="sr-only d-md-none">{{ 'previous'|trans|ucfirst }}</span><span class="d-none d-md-inline-block">{{ 'previous'|trans|ucfirst }}</span></a>
+        <a href="/" data-button-previous="back" class="breadcrumb-link-back"><i
```

Comment:
Is het niet logischer om het i-element ook op een nieuwe lijn te zetten? Ik vind dat leesbaarder

Reply:
Zeker en vast, fixed in 7f3cf7a6e7477cce0bc97ab7c56cbf2af97e97b8